### ISSUE TITLE
Attribute map collisions 

### ISSUE THREAD

**Issue by c00kiemon5ter** [rohe/pysaml2#489]
Looking into [attributemaps/saml_uri.py][amsu] I see the following (picking parts):

```py
UCL_DIR_PILOT = 'urn:oid:0.9.2342.19200300.100.1.'
LDAPGVAT_UCL_DIR_PILOT = UCL_DIR_PILOT

MAP = {
# [...]
        LDAPGVAT_UCL_DIR_PILOT+'1': 'PVP-USERID',
        LDAPGVAT_UCL_DIR_PILOT+'3': 'PVP-MAIL',
# [...]
        UCL_DIR_PILOT+'1': 'uid',
        UCL_DIR_PILOT+'3': 'mail',
# [...]
}
```

This will of course result in `PVP-USERID` and `PVP-MAIL` not exist as values in the dict, as they will be overwritten by `uid` and `mail` respectively. 

  [amsu]: https://github.com/rohe/pysaml2/blob/master/src/saml2/attributemaps/saml_uri.py

**Comment by peppelinux** [comment]
Yes this is so, faced with R&S and SPID profile together in SATOSA, with attr name format both in basic and oid.
collisions!
